tiled/server/file_response_with_range.py

```diff
@@ -0,0 +1,96 @@
+# This is adapted from a closed PR in starlette which was reviewed by
+# a core maintainer but put aside for now in favor of other priorities.
```

Comment:
Perhaps not put aside anymore?

Reply:
Ah, I meant put aside by starlette's core devs, not by us. I revised the comment to clarify and add a little more  context.